src/main/java/com/vackosar/gitflowincrementalbuild/control/Property.java

```diff
@@ -5,21 +5,26 @@
 
 public enum Property {
     enabled("true"),
-    repositorySshKey(""),
```

Comment:
So this was replaced by proper configuration of git?

Reply:
I think I added this property for certain use case there. I think there was some reason that it was needed in Teamcity. But I don't remember exactly why. Since nobody complains that this is not there, I guess we are fine without it.